Mage/src/main/java/mage/abilities/common/delayed/CopyNextSpellDelayedTriggeredAbility.java

```diff
@@ -0,0 +1,52 @@
+package mage.abilities.common.delayed;
+
+import mage.abilities.DelayedTriggeredAbility;
+import mage.abilities.effects.common.CopyTargetSpellEffect;
+import mage.constants.Duration;
+import mage.game.Game;
+import mage.game.events.GameEvent;
+import mage.game.stack.Spell;
+import mage.target.targetpointer.FixedTarget;
+
+/**
+ * @author TheElk801
+ */
+public class CopyNextSpellDelayedTriggeredAbility extends DelayedTriggeredAbility {
```

Comment:
I think this should be ranamed, or to take in a filter to specify what kind of next spell.

It's currently named `CopyNextSpellDelayedTriggeredAbility` but it really is `CopyNextInstanrOrSorcerySpellDelayedTriggeredAbility`.

Reply:
yeah I was thinking about adding the filter but was too lazy to do it, I'll add it